src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, ReactNode, useMemo } from 'react'
+import { VariantProps, tv } from 'tailwind-variants'
+
+export type Props = {
+  label: ReactNode
+  size?: VariantProps<typeof chip>['size']
+  className?: string
+}
+
+export const chip = tv({
```

Comment:
スタイリングの export も外から使う場面が出たタイミングで考えたい。

```suggestion
const chip = tv({
```